lib/experiments/index.js

```diff
@@ -1,7 +1,17 @@
 'use strict';
 var symbol = require('../utilities/symbol');
+var version = require('../../package').version;
+var isCanaryVersion = version.match(/-alpha/);
```

Comment:
The basic idea is to ensure that the experiments present on master are only available when using "canary" builds.

#6576 enables us to use version numbers that haven't been published to NPM yet (which is the only real reason that master uses the latest beta version number anyways).

If we want to use some other mechanism to accomplish this, I think we could. This just seems like the easiest/best path forward to me.

Happy to add to next meeting agenda to discuss more in depth...